libcxx/test/std/time/time.clock/time.clock.system/sys_time.ostream.verify.cpp

```diff
@@ -0,0 +1,74 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: no-localization
+// UNSUPPORTED: GCC-ALWAYS_INLINE-FIXME
+
+// TODO FMT This test should not require std::to_chars(floating-point)
+// XFAIL: availability-fp_to_chars-missing
+
+// <chrono>
+
+// class system_clock;
+
+// template<class charT, class traits, class Duration>
```

Comment:
Same comment here for using SFINAE. I see that for a few expressions, you ensure that they work instead of *not* working. For those, we should also test them positively in `static_assert(is_ostreamable<expression>)`.

Reply:
I'll SFINAE these, this function is indeed constrained.